src/Init/Data/BitVec/Basic.lean

```diff
@@ -475,20 +475,26 @@ def shiftLeftZeroExtend (msbs : BitVec w) (m : Nat) : BitVec (w + m) :=
 /--
 Zero extend vector `x` of length `w` by adding zeros in the high bits until it has length `v`.
 If `v < w` then it truncates the high bits instead.
```

Comment:
I'd suggest we rephrase the doc a bit, to make sure we don't emphasize zeroExtension over truncation

```suggestion
/--
Transform `x` of length `w` into a bitvector of length `v`, by either:
- zero extending, that is, adding zeros in the high bits until it has length `v`, if `v > w`, or
- truncating the high bits, if `v < w`.
```